### ISSUE TITLE
New Content Update Request: Add ATX events

### ISSUE THREAD

**Issue by fvntr** [techqueria/website#506]
## Link where content currently exists

there are some atx events that are still not reflected 

## Updates you'd like to make

Add these events to the events list: 
https://www.eventbrite.com/o/darlene-santoyo-andrea-cardona-18541630187

**Comment by apsychogirl** [comment]
Hi, Can I take this up ?

**Comment by fvcproductions** [comment]
@apsychogirl yep sure can

**Comment by apsychogirl** [comment]
Hi, I had a little issue in finding the file where I need to update the list. Can you please help?

**Comment by fvcproductions** [comment]
Yeah you can check out https://github.com/techqueria/website/pull/557 for an example of how to add a new event @apsychogirl
